### ISSUE TITLE
Pension | Fix header on confirmation page

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#107545]
## Issue Description

The Frontend CoP team will be updating Cypress Axe tests to;
- Ensure page headers aren't empty
- Check the appropriate order of headers on a page
- Ensure every page includes a header

A preliminary run with these new rules enabled indicated an issue with the [Pensions](https://github.com/department-of-veterans-affairs/vets-website/actions/runs/13770651994/job/38508967544) (simple & kitchen sink)

After running the e2e test locally, it appears that the failure is from the confirmation page header level order - an `H4` following an `H2`

## Acceptance Criteria
- [ ] Fix header order level
- [ ] Ensure E2E & axe tests are passing
- [ ] Review all e2e test with new axe checks enabled

## Definition of Done
- [ ] All tests passing
- [ ] Axe checks all passing
